backend/src/main/java/com/htecgroup/skynest/service/impl/BucketServiceImpl.java

```diff
@@ -98,6 +99,13 @@ public List<BucketResponse> listAllDeletedBuckets() {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public void activateLambda(UUID bucketId, LambdaType lambdaType) {
+    BucketEntity bucketEntity = findBucketEntityById(bucketId);
+    bucketEntity.addLambda(lambdaType);
+    bucketRepository.save(bucketEntity);
```

Comment:
Did you try the code without the save method?

Reply:
I tried it, but it's not saving the added lambda in the database.

Reply:
ok, maybe a transaction manager is needed, but leave it like that